Source/Core/DolphinQt/Settings/InterfacePane.cpp

```diff
@@ -217,12 +219,12 @@ void InterfacePane::LoadConfig()
   m_combobox_theme->setCurrentIndex(
       m_combobox_theme->findText(QString::fromStdString(SConfig::GetInstance().theme_name)));
 
-  const QString userstyle = Settings::Instance().GetCurrentUserStyle();
+  const QFileInfo userstyle = Settings::Instance().GetCurrentUserStyle();
 
-  if (userstyle.isEmpty())
+  if (userstyle.fileName().isEmpty())
     m_combobox_userstyle->setCurrentIndex(0);
   else
-    m_combobox_userstyle->setCurrentText(userstyle);
+    m_combobox_userstyle->setCurrentIndex(m_combobox_userstyle->findText(userstyle.baseName()));
```

Comment:
You may as well move this up and then check if ``findText``s return value is -1 or not

Reply:
Given that there's no reason to change the type of ``userstyle`` so you can just use ``QFileInfo(userstyle).baseName()`` once.

Reply:
I'm not really sure how exactly you want to see this changed

```
  const QString userstyle = Settings::Instance().GetCurrentUserStyle();
  const int index = m_combobox_userstyle->findText(QFileInfo(userstyle).baseName());

  if (index > 0)
    m_combobox_userstyle->setCurrentIndex(index);
```
Would this be fine?

Reply:
Yes.